### PR TITLE
Feature/crossword

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: moj-node/server/routes/games.js [hunk 1]
```diff
@@ -120,6 +120,25 @@ const createGamesRouter = ({ analyticsService, logger }) => {
...
+  router.get('/crossword', (req, res) => {
+    logger.info('GET /games/crossword');
+    const userDetails = path(['session', 'user'], req);
+    const newDesigns = path(['locals', 'features', 'newDesigns'], res);
+    const sessionId = path(['session', 'id'], req);
+
+    config.newDesigns = newDesigns;
+    config.detailsType = 'small';
+    config.userName = path(['name'], userDetails);
+
```
Comment (psoleckimoj) [inline moj-node/server/routes/games.js:132]: GA stuff needed :D

File: moj-node/server/views/pages/games/crossword.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
+{% from "../../components/feedback-widget/macro.njk" import hubFeedbackWidget %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "../../components/page-navigation/macro.njk" import pageNavigation %}
+{% from "../../components/user-details/macro.njk" import userDetails %}
+{% extends "../../components/template.njk" %}
+
+{% block pageTitle %}
+  {{ data.title }}
```
Comment (psoleckimoj) [inline moj-node/server/views/pages/games/crossword.html:9]: Can also tidy this to just `title`

File: moj-node/server/routes/games.js (hunk 1 above)
Comment (psoleckimoj) [reply moj-node/server/routes/games.js]: Done

File: moj-node/server/views/pages/games/crossword.html (hunk 2 above)
Comment (psoleckimoj) [reply moj-node/server/views/pages/games/crossword.html]: Done

File: moj-node/server/routes/games.js (hunk 1 above)
Comment (LucasCairns) [reply moj-node/server/routes/games.js]: Added the tracking event

File: moj-node/server/views/pages/games/crossword.html (hunk 2 above)
Comment (LucasCairns) [reply moj-node/server/views/pages/games/crossword.html]: Updated the template

**Pull request merged**